pages/api/csv-update.page.ts

```diff
@@ -78,9 +78,12 @@ const updateCsvData = async (
       return;
     }
 
-    const {
-      fields: { accountListId, csvFileId, initialData, patchedData, include },
-    } = await parseBody(req);
+    const { fields } = await parseBody(req);
+    const accountListId = fields.accountListId?.[0];
```

Comment:
It's interesting that it has switched to return an array for each item.

Reply:
I know. It seems that instead of it being either a string/file or an array of strings/files, they decided to make it always be an array.